google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/LegacySQLTypeName.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigquery;
+
+/**
+ * A type used in legacy SQL contexts. For example, these types are used in query results,
+ * even if the query is a standard SQL query.
```

Comment:
The wording here isn't the best - I will make it clearer. I will also add the `@see` link.